projects/demo-playwright/tests/addon-commerce/input-card-grouped.spec.ts

```diff
@@ -0,0 +1,165 @@
+import {
+    TuiDocumentationPagePO,
+    tuiGoto,
+    TuiInputCardGroupedPO,
+} from '@demo-playwright/utils';
+import {expect, Locator, test} from '@playwright/test';
+
+const {describe, beforeEach, use} = test;
+
+describe(`InputCardGrouped`, () => {
+    use({viewport: {width: 1280, height: 800}});
```

Comment:
Let use
```ts
use({viewport: {width: 700, height: 800}});
```

for `describe('API')`

___

And use

```ts
use({viewport: {width: 1280, height: 800}});
```

for `describe('Examples')`